cmd/notation/sign.go

```diff
@@ -35,6 +35,28 @@ func signCommand(opts *signOpts) *cobra.Command {
 	command := &cobra.Command{
 		Use:   "sign [reference]",
 		Short: "Signs artifacts",
+		Long: `
+** This command is in preview and under development. **
+Prerequiste: a signing key needs to be configured using the command "notation key".
+
+Example - Sign a container image using the default signing key, with the default JWS envelope:
+  notation sign <registry>/<repository>:<tag>
+
+Example - Sign a container image using the default signing key, with the COSE envelope:
+  notation sign <registry>/<repository>:<tag> --envelope-type cose
```

Comment:
Let's make the flags before the arguments.

```suggestion
  notation sign --envelope-type cose <registry>/<repository>:<tag>
```

Reply:
Good catch. Update.